server/container/core/src/main/java/org/apache/james/server/core/filesystem/ResourceFactory.java

```diff
@@ -59,15 +57,15 @@ public Resource getResource(String fileURL) {
             try {
                 // Try to parse the location as a URL...
                 return handleUrlResource(fileURL);
-            } catch (MalformedURLException | URISyntaxException ex) {
+            } catch (MalformedURLException ex) {
                 // No URL -> resolve as resource path.
                 return new ClassPathResource(fileURL);
             }
         }
     }
 
-    private Resource handleUrlResource(String fileURL) throws MalformedURLException, URISyntaxException {
-        URL url = new URI(fileURL).toURL();
+    private Resource handleUrlResource(String fileURL) throws MalformedURLException {
+        URL url = new URL(fileURL);
```

Comment:
the logic for handler UrlResource is more complicated than I think -> I revert it now